### PR TITLE
CSVS-112 Integration tests for token creation/deleting were implemented

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import com.bwsw.cloudstack.vault.server.common.Converter
+import com.bwsw.cloudstack.vault.server.util.vault.TokenData
+import com.bwsw.cloudstack.vault.server.util.{IntegrationTestsSettings, RequestPath}
+import com.bwsw.cloudstack.vault.server.vault.entities.Policy
+import org.scalatest.FlatSpec
+
+class VaultServiceIntegrationTestSuite extends FlatSpec with IntegrationTestsComponents {
+  val mapper = new JsonMapper(ignoreUnknownProperties = true)
+
+  it should "create token and then revoke it" in {
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala]: replace "it" by a class name

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    val policiesFromRevokedToken = vaultService.revokeToken(token)
+
+    assert(policiesFromRevokedToken.toSet == Set(policy.name))
+
+    val responseRevokedToken = new Rest()
+      .url(s"${vaultService.endpoint}${RequestPath.vaultTokenLookup}")
+      .header("X-Vault-Token", IntegrationTestsSettings.vaultRootToken)
+      .body(jsonTokenId.getBytes("UTF-8")).post()
+
+    assert(responseRevokedToken.getStatus == 403)
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala]: add a note of the status meaning or wrap the status in a variable

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala]: ok

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala]: ok

**Review by placccebo — APPROVED** [review]

**Comment by MedvedevBW** [comment]
[run tests]

**Pull request merged**